op-challenger/cmd/watch/cmd.go

```diff
@@ -0,0 +1,35 @@
+package watch
+
+import (
+	"github.com/urfave/cli"
+
+	"github.com/ethereum-optimism/optimism/op-challenger/config"
+)
+
+var Subcommands = cli.Commands{
+	{
+		Name:  "oracle",
+		Usage: "Watches the L2OutputOracle for new output proposals",
+		Action: func(ctx *cli.Context) error {
+			logger, err := config.LoggerFromCLI(ctx)
+			if err != nil {
+				return err
+			}
+			logger.Info("Listening for new output proposals")
+
+			cfg, err := config.NewConfigFromCLI(ctx)
+			if err != nil {
+				return err
+			}
+
+			return Oracle(logger, cfg)
+		},
+	},
+	{
+		Name:  "factory",
+		Usage: "Watches the DisputeGameFactory for new dispute games",
+		Action: func(ctx *cli.Context) error {
+			panic("factory not implemented")
+		},
```

Comment:
I probably just wouldn't introduce this until the next PR where it gets implemented rather than panicing.